### PR TITLE
Fixing the bug that appeared when anonymize customer last name option has been added.

### PR DESCRIPTION
**Pull request by Oksydan**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Fixing the bug that appeared when anonymize customer last name option has been added.
| Type?         | bug fix
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes PrestaShop/Prestashop#22222.
| How to test?  | Issue describes it enough.

### CODE REVIEW

File: controllers/front/ListComments.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public function display()
             $productComment['content'] = htmlentities($productComment['content']);
             $productComment['date_add'] = $dateFormatter->format($dateAdd);
 
-            if ($isLastNameAnynomus) {
+            if ($isLastNameAnynomus && isset($productComment['firstname']) && isset($productComment['lastname'])) {
```
Comment (NeOMakinG) [inline controllers/front/ListComments.php:68]: Hello

What's the idea behind checking if firstname is set if we don't use it in the condition ?

**Comment by NeOMakinG** [comment]
CSFixer is red, please run it :)

File: controllers/front/ListComments.php (hunk 1 above)
Comment (Oksydan) [reply controllers/front/ListComments.php:68]: Hi, 
`isset($productComment['firstname']) && isset($productComment['lastname'])` - for deleting account firstname and lastname is not set. I just want to be sure that nothing weird are going to happen when first name will not be available.

Comment (Oksydan) [reply controllers/front/ListComments.php:68]: Well, you are right. Check exists on line 74.

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
Please rebase your pull request

**Comment by Oksydan** [comment]
@PierreRambaud I did it against latest dev branch with your CS fixer changes.

File: controllers/front/ListComments.php [hunk 2]
```diff
@@ -65,7 +65,7 @@ public function display()
             $productComment['content'] = htmlentities($productComment['content']);
             $productComment['date_add'] = $dateFormatter->format($dateAdd);
 
-            if ($isLastNameAnynomus) {
+            if ($isLastNameAnynomus && isset($productComment['firstname'])) {
```
Comment (atomiix) [inline controllers/front/ListComments.php:68]:
```suggestion
            if ($isLastNameAnynomus && isset($productComment['lastname'])) {
```

Comment (Oksydan) [reply controllers/front/ListComments.php]: @atomiix 
thank you, my bad 😞

**Comment by atomiix** [comment]
@Oksydan I don't know how you rebased your PR but something's wrong 🤔, you shouldn't see someone else's commits on your PR.

**Comment by Oksydan** [comment]
@atomiix 
branch fixed

**Review by PierreRambaud — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by khouloudbelguith** [comment]
Hi @Oksydan,

It is ok :heavy_check_mark: 
image

Thanks!

**Comment by PierreRambaud** [comment]
Thank you @Oksydan

**Pull request merged**